src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -159,6 +159,7 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
     >["suggestedAddresses"])
   const inputAddress = verifyAddress.inputAddress as AddressVerificationFlow_verifyAddress$data["inputAddress"]
   const verificationStatus = verifyAddress.verificationStatus as AddressVerificationFlow_verifyAddress$data["verificationStatus"]
+  const userId = user?.id
```

Comment:
This would still change on every render. I'm surprised that this doesn't manifest the bug in the spec but this wouldn't fix it if it's an actual bug.

Reply:
If you wanted to do this you would use a ref. But that's probably going to still be a bug.

Better would be just to remove these effects.

Reply:
My theory here was that for some reason the `user` object is changing identity and because of how the dependency array evaluates objects the effect is being triggered again. I've only seen this in tests, so could it be part of our test setup?

Reply:
Yeah, it's definitely part of the test setup. It's unclear to me whether or not this is something that's possible to drive to in the browser — though that's sort of why I think you should just get rid of the effects: easier to reason about.

Reply:
I agree w/ you @dzucconi but think that removing the useEffects() should go in a follow up PR.